### ISSUE TITLE
Scrape multiple things a from a single HTML node with custom conversion

### ISSUE THREAD

**Issue by ComFreek** [IonicaBizau/scrape-it#77]
Let's say we have the HTML:
```html
<table>
  <tr>
    <td>21.11.2017<br>22.11.2017</td>
    <td>Event A</td>
  </tr>
 <tr>
    <td>25.11.2017<br>27.11.2017</td>
    <td>Event B</td>
  </tr>
</table>
```

I would like to parse every row's first cell as two entries in my object: `startDate` and `endDate`.
Currently, I doing this redundantly:

```js
function scrapeDate(first, str) {
  /* Redundant code section */
  let myData = str.split(' '); // or regex
  /* ... */
  return first ? myData[0] : myData[1];
}

scrapeIt(URI, {
  events: {
    listItem: "table tr"
    data: {
      startDate: {
        selector: "td",
        eq: 0,
        convert: scrapeDate.bind(null, true)
      },
      endDate: {
        selector: "td",
        eq: 0,
        convert: scrapeDate.bind(null, false)
      }
  }
});
```

Just opening this here, maybe I get an idea on how a flexible but clean API could look like.

**Comment by IonicaBizau** [comment]
I think this is similar to #45. How does `td:eq(1)` work at all in your example? That's going to select the second column, right?

```js
scrapeIt(URI, {
  events: {
    listItem: "table tr"
    data: {
      dates: {
        selector: "td",
        eq: 0,
        convert (v) {
          v = v.split(" ")
          v.startDate = v[0]
          v.endDate = v[1]
          return v
        }
     }
  }
});
```

Maybe this looks better. I didn't test it.

**Comment by ComFreek** [comment]
Right, I meant `td:eq(0)` in both cases. This is now corrected in the post above.

Indeed, this is a much cleaner solution! At least in those cases where the website puts related data inside the same node. In the cases where it doesn't (e.g. `<td>` contains a date and possibly flag 'only with invitation'), it forces the programmer to put them into the same JSON tree as well.

**Comment by IonicaBizau** [comment]
Fixed via #76. Thanks, @ComFreek!
